src/ui/screens/settings/ViewPreferences.qml

```diff
@@ -56,6 +69,14 @@ MZViewBase {
             isChecked: MZSettings.gleanEnabled
             onClicked: {
                 MZSettings.gleanEnabled = !MZSettings.gleanEnabled
+                if (MZSettings.gleanEnabled)
+                {
+                    Glean.interaction.dataCollectionEnabled.record({screen:telemetryScreenId})
+                }
+                else
+                {
+                    Glean.interaction.dataCollectionDisabled.record({screen:telemetryScreenId})
+                }
```

Comment:
This metric makes no sense. It will always be `true`, when data collection is disabled `record` is a no-op.

Reply:
I just followed the Miro board I was given and I assumed this means we would still send one last metric saying they opted out but if that's not the case then yeah there's no reason this needs to be recorded

Reply:
The metic will be sent, however when users opt out of telemetry their data gets deleted from our servers in up to 30 days. Would you mind explaning this to Santiago? I'd rather we don't merge this metric since it doesn't really make sense.